PennMobile/src/main/res/layout/dining_hall_widget_item.xml

```diff
@@ -0,0 +1,57 @@
+
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/widgetBackground"
+    android:layout_width = "203dp"
+    android:layout_height = "220dp"
+    android:theme="@style/Theme.Pennmobileandroid.AppWidgetContainer">
+
+    <ImageView
+        android:id="@+id/diningBackground"
+        android:background="@drawable/roundcorner"
+        android:clipToOutline="true"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:foreground="@drawable/white_fade_gradient"
+        android:scaleType="centerCrop"
+        android:src="@drawable/dining_commons" />
+
+    <TextView
+        android:id="@+id/appwidget_text"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_alignParentBottom="true"
+        android:layout_marginBottom="8dp"
+        android:contentDescription="@string/appwidget_text"
+        android:text="Lauder\nCollege House"
+        android:layout_marginLeft="10dp"
+        android:textColor="@color/black"
+        android:textSize="22sp"
+        android:textStyle="bold" />
+
+    <TextView
+        style="@style/DiningStatusLabel"
+        android:id="@+id/textView3"
```

Comment:
Definitely need to rename this, but why textView3??? What happened to textView1 and textView2????